dlpy/model.py

```diff
@@ -3375,4 +3396,168 @@ def parse_results(response, connection, userdata):
             print(response.disposition.status)
             print(response.disposition.debug)
 
-    return parse_results
```

Comment:
this even can its own file

Reply:
Creating tensorboard.py and test_tensorboard.py